README.md

```diff
@@ -1364,9 +1364,19 @@ Valid options: `true` and `false`.
 
 Default value: `true`.
 
-##### `environment`
+##### `proxy_server`
 
-Environment variables for settings such as http_proxy, https_proxy, or ftp_proxy. These are passed through to the staging module and then to the underlying exec(s), so it follows the same format of the exec type [`environment`](https://docs.puppet.com/puppet/latest/reference/type.html#exec-attribute-environment).
+Specify a proxy server to use when downloading tomcat binaries. e.g.: `https://example.com:8080`
+
+Default value: `undef`.
+
+##### `proxy_type`
+
+Specify the proxy server type used by `proxy_server`. Normally this defaults to the protocol specified in the `proxy_server` URI
```

Comment:
* Change "Specify" to "Specifies"
* Needs a period at the end of the second sentence.